gherkin/javascript/test/GherkinLineTest.ts

```diff
@@ -0,0 +1,28 @@
+import * as assert from 'assert'
+import GherkinLine from '../src/GherkinLine'
+
+describe('GherkinLine', () => {
+  describe('#getTableCells', () => {
+    function getCellsText(line: string) {
+      const gl = new GherkinLine(line, 1)
+
+      return gl.getTableCells().map(span => span.text)
+    }
+
+    it('trims white spaces before cell content', () => {
+      assert.deepEqual(getCellsText("|   \t spaces before|"), ['spaces before'])
+    })
+
+    it('trims white spaces after cell content', () => {
+      assert.deepEqual(getCellsText("|spaces after   |"), ['spaces after'])
+    })
+
+    it('trims white spaces around cell content', () => {
+      assert.deepEqual(getCellsText("|   \t spaces everywhere   \t|"), ['spaces everywhere'])
+    })
+
+    it('does not delete white spaces inside a cell', () => {
```

Comment:
not super important. But you could maybe rename this to 

`it does not delete white spaces in the middle of cell content`